docs/rackhd/event_notification.rst

```diff
@@ -1,105 +1,272 @@
 Event Notification
 ------------------
 
-The change of resources managed by RackHD could be retrieved from AMQP messages.
-
-- Exchange: **on.events**
-- Routing Key **event.<type>**
+RackHD support event notification via both AMQP and web hook.
 
-`<type>` is event's type, it's the same with event's payload `type`, see payloads_ for more details.
+A web hook allows an application to subscribe certain RackHD published events published by configured URL, when one of the subscribed events is triggered, RackHD will send a HTTP POST request with event payload to configured URL.
```

Comment:
"subscribe certain RackHD published events published by configured URL". a little strange to have two "publish" in this sentence